spec/import-js/mock_vim_buffer.rb

```diff
@@ -1,26 +1,37 @@
+# Class that is used in tests to behave similarly to the VIM::Buffer object.
+# @see http://vimdoc.sourceforge.net/htmldoc/if_ruby.html#ruby-buffer
 class MockVimBuffer
+  # @param buffer_text [String]
   def initialize(buffer_text)
     @buffer_lines = buffer_text.split("\n")
   end
 
+  # @param one_indexed_n [Number]
   def delete(one_indexed_n)
     @buffer_lines.delete_at(one_indexed_n - 1)
   end
 
+  # @param index [Number] line number to append after
+  # @param string [String] line to append
   def append(index, string)
     # We replace newlines with "^@" because that's what a real vim buffer will
     # output if you append such a string.
     @buffer_lines.insert(index, string.gsub("\n", '^@'))
   end
 
+  alias_method :length, :count
```

Comment:
Yes. My first instinct was to put it after the method, but I decided to put it before because of the example at http://apidock.com/ruby/Module/alias_method

I will force push a new version in a moment.